### PR TITLE
Use footcite in evoked.py

### LINKED ISSUE #8935: use bibtex in viz/evoked.py
convert references in `mne/viz/evoked.py` to use footcite / footbibliography

### PR DESCRIPTION
**Pull request by mdclarke**
#### Reference issue
missing bibtex, closes #8935

#### What does this implement/fix?
added footcite reference

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1036,9 +1036,10 @@ def plot_evoked_white(evoked, noise_cov, show=True, rank=None, time_unit='s',
     """Plot whitened evoked response.
 
     Plots the whitened evoked response and the whitened GFP as described in
-    [1]_. This function is especially useful for investigating noise
-    covariance properties to determine if data are properly whitened (e.g.,
-    achieving expected values in line with model assumptions, see Notes below).
+    :footcite:`EngemannGramfort2015'. This function is especially useful for
```
Comment (sappelhoff) [inline mne/viz/evoked.py:1039]: The "footcite" directive should be used like this:

```
:footcite:`SomeKeyToCite`
```

You have a typo there:
```suggestion
    :footcite:`EngemannGramfort2015`. This function is especially useful for
```

**Comment by sappelhoff** [comment]
> missing bibtex

could you point to the issue number that this is fixing?

- go to https://github.com/mne-tools/mne-python/issues
- find the "issue" that you wanted to fix, and mark down its number
- edit the first comment in this pull request to include the phrase `closes #issue-number` where issue-number should be replaced with the number you found

That is important for crosslinking and to show other people *what* you are working on.

Finally: You could also consider to improve your PR title. One example: `Use footcite in evoked.py` --> this would tell reviewers more about the scope of your PR than `Add bibtex` :slightly_smiling_face:

**Comment by mdclarke** [comment]
> > missing bibtex
> 
> could you point to the issue number that this is fixing?
> 
> * go to https://github.com/mne-tools/mne-python/issues
> * find the "issue" that you wanted to fix, and mark down its number
> * edit the first comment in this pull request to include the phrase `closes #issue-number` where issue-number should be replaced with the number you found
> 
> That is important for crosslinking and to show other people _what_ you are working on.
> 
> Finally: You could also consider to improve your PR title. One example: `Use footcite in evoked.py` --> this would tell reviewers more about the scope of your PR than `Add bibtex` 🙂

Done! Thanks for your help

**Comment by sappelhoff** [comment]
:+1: one more thing --> look closely at what you wrote: `closes issue #8935`

GitHub as an automatic parser that detects things like `closes #8935` or `fixed #8935` --> **WITHOUT** the `issue` in between the verb and the number.

If GitHub detects that, it automatically closes the issue once the associated PR is merged --> neat! :wink: 

So please go back and edit once more so that it says `closes #8935`, then (after saving the edit) try to hover over that part of the text ... and GitHub will tell you exactly what I just wrote.

**Pull request merged**